src/index.css

```
@@ -1,10 +1,18 @@
-/*======================<Pretendard Font>/*======================*/
+/*======================<React flow custom edge>======================*/
+.button-edge__label {
+  position: absolute;
+  pointer-events: all;
+  z-index: 10;
+}
+/*======================<React flow custom edge>======================*/
+
+/*======================<Pretendard Font>======================*/
 @import url('/fonts/pretendard/pretendardvariable.css');
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)